clang/lib/AST/Type.cpp

```diff
@@ -94,6 +94,8 @@ bool Qualifiers::isTargetAddressSpaceSupersetOf(LangAS A, LangAS B,
          (A == LangAS::Default &&
           (B == LangAS::cuda_constant || B == LangAS::cuda_device ||
            B == LangAS::cuda_shared)) ||
+         // Default is a superset of HLSL private.
+         (A == LangAS::Default && B == LangAS::hlsl_private) ||
```

Comment:
Is default actually a superset, or is this a hack that must be fixed up later?

Reply:
Right, need to update the comment.